packages/input/__tests__/input.spec.ts

```diff
@@ -2,6 +2,7 @@ import { ref } from 'vue'
 import { mount } from '@vue/test-utils'
 import { sleep, defineGetter } from '@element-plus/test-utils'
 import Input from '../src/index.vue'
+import input from '..'
```

Comment:
What is this for?

Reply:
Remove this unused code.

Reply:
Done. It should have been imported automatically by accident.